deepcell_tracking/tracking.py

```diff
@@ -421,349 +614,192 @@ def _update_tracks(self, assignments, frame, predictions):
                 y_tracked_update[self.y[[frame]] == new_label] = new_track_id + 1
                 self.y[frame][self.y[frame] == new_label] = new_track_id + 1
 
-            # Dont touch anything if there was a cell that "died"
-            elif track < number_of_tracks and cell > number_of_cells - 1:
-                continue
+        # Check and make sure cells that divided did not get assigned to the same cell
+        for track in range(len(self.tracks)):
```

Comment:
do we need to cycle over all tracks here to provide caps? I think we could do this by assignment at time of division